### PR TITLE
Adds X509Certificate Pack for Certificate Indicator Type

### PR DESCRIPTION
**Pull request by jtschichold**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
*This PR depends on PR https://github.com/demisto/content/pull/10234*

This pack adds the Certificate Indicator Type representing X509 Certificates.
Contents:
- Certificate Indicator Type with mapper
- Indicatore Fields for Certificate Indicator
- Layout for Certificate Indicator
- A script to extract all the certificate fields from a X509 certificate in PEM or DER format (`CertificateExtract`)
- A reputation script to calculate the DBot Score of a Certificate Indicator based on some simple validation checks
- Test Playbook

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/X509Certificate/Scripts/CertificateReputation/CertificateReputation.yml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  type: Number
+script: '-'
+system: false
+tags:
+  - reputation
+timeout: '0'
+type: python
+subtype: python3
+dockerimage: demisto/python3:3.8.6.13358
+fromversion: 5.5.0
```
Comment (Bargenish) [inline Packs/X509Certificate/Scripts/CertificateReputation/CertificateReputation.yml:145]:
```suggestion
fromversion: 6.0.0
```

File: Packs/X509Certificate/TestPlaybooks/playbook-X509Certificate_Test_Playbook.yml [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+        "height": 955,
+        "width": 380,
+        "x": 450,
+        "y": 50
+      }
+    }
+  }
+inputs: []
+outputs: []
+fromversion: 5.5.0
```
Comment (Bargenish) [inline Packs/X509Certificate/TestPlaybooks/playbook-X509Certificate_Test_Playbook.yml:403]:
```suggestion
fromversion: 6.0.0
```

**Review by Bargenish — COMMENTED** [review]
@jtschichold This looks great!

Few small notes:

* Entities in a pack should have the same `fromversion` (I've suggested `6.0.0`, change as needed)
* Please rename the `Certificate` indicator file from `Packs/X509Certificate/IndicatorTypes/reputation-58cb0f03-3b63-4832-8a79-029c09535bbe-Certificate.json`
to `Packs/X509Certificate/IndicatorTypes/reputation-Certificate.json`
* Add docstrings to the more complex functions in the `CertificateExtract` script.
* Please update @altmannyarden and me once https://github.com/demisto/content/pull/10234 was merged in order to schedule a demo.

**Comment by fvigo** [comment]
Merged from `upstream/master` as we had some dependencies on CommonServerPython Base pack 1.4.1 we needed

**Comment by jtschichold** [comment]
@Bargenish @altmannyarden Pushed the changes requested in the review. The contrib branch should be updated to the latest master but I don't have write access.

**Review by Bargenish — APPROVED** [review]

**Pull request merged**